### PR TITLE
Mixins

### PR DESCRIPTION
**Pull request by owennw**
Adds a size mixin, and a tiny refactor to the overflow CSS in the minichart.

Note I've not included the size mixin in the showcase overrides.

### CODE REVIEW

File: src/sidebars/sidebar.less [hunk 1]
```diff
@@ -2,28 +2,26 @@
 @flash-colour: #447272;
 @bottom-tab-height: 40px;
 
-@-webkit-keyframes search {
-  0% {
+.click-flash(@final-colour) {
+    0% {
```
Comment (WPFerg) [inline src/sidebars/sidebar.less]: Indentation

**Comment by WPFerg** [comment]
One small whitespace comment but looks fine to me. I'm guessing you can't do something like this:

```less
.class {
  .size(@width: 10px, @height:10px);

  &.some-other-class {
    .size(@height: 10px);
  }
}
```
because `@width` is overridden? (Like [here](https://github.com/ScottLogic/StockFlux/pull/699/files#diff-da44fe9f3c295da97d70c9a6a6ced0adR6)?)

**Comment by owennw** [comment]
Yes that is the case at the moment, which is maybe not the best. Any ideas on how to improve that?

**Comment by WPFerg** [comment]
Theres [guards](http://lesscss.org/features/#css-guards-feature) but I'm not sure if they'd work for something like `.size( ... ) when (@width)`

File: src/sidebars/favourites/minichart/minichart.less [hunk 2]
```diff
@@ -1,19 +1,22 @@
 ﻿@minichart-error-colour: white;
 
+.overflow {
+  overflow: visible;
```
Comment (WPFerg) [inline src/sidebars/favourites/minichart/minichart.less]: Trying it out shows some suspicious line endings here

**Comment by owennw** [comment]
Updated the commits now.

**Pull request merged**